packages/main/src/Form.ts

```diff
@@ -365,10 +366,15 @@ class Form extends UI5Element {
 		return this.hasCustomHeader ? undefined : `${this._id}-header-text`;
 	}
 
+	get ariaRole(): string | undefined {
```

Comment:
rename to effectiveAccessibleRole in order to not conflict with the native ariaRole typescript definition.

Reply:
done